README.md

```diff
@@ -255,6 +277,9 @@ You can change the default distro by running
 wsl --setdefault <process name>
 ```
 
-After it finishes changing, open your Docker Desktop and navigate `Settings > Resources > WSL Integration`, make sure you check "Enable integration with my default WSL distro", and enable integrations with any of the additional distros you have. You'll have to restart your docker by doing this operation.
+After it finishes changing, open your Docker Desktop and enable `Use the WSL 2 based engine`. Then navigate to `Settings > Resources > WSL Integration`, make sure you check `Enable integration with my default WSL distro`, and enable integrations with any of the additional distros you have. You'll have to restart your docker by doing this operation.
+
+#### Docker Desktop Setting Screenshots
 
-![Docker Desktop WSL image](assets/docker-wsl.png)
+![Docker Desktop General Settings](assets/docker-home.png)
+![Docker Desktop Resources Settings](assets/docker-wsl.png)
```

Comment:
Did you mean to remove this line and/or add the `docker-wsl.png` to the commit? I think it's missing from the PR somehow? 

![image](https://user-images.githubusercontent.com/17013462/118081321-f77f7800-b36f-11eb-82bd-78e538f07a72.png)

Reply:
Nevermind, resolved offline because I cannot see what's in front of me. 🙈